src/time/tests/PiecewiseWaveformTest.cpp

```diff
@@ -0,0 +1,306 @@
+#include <Eigen/Core>
+#include "testing/TestContext.hpp"
+#include "testing/Testing.hpp"
+#include "testing/WaveformFixture.hpp"
+#include "time/Time.hpp"
+#include "time/Waveform.hpp"
+
+using namespace precice;
+using namespace precice::time;
+
+BOOST_AUTO_TEST_SUITE(TimeTests)
+// @TODO rename to SubcyclingWaveformTests?
```

Comment:
```suggestion
```

Reply:
I merged this file with `WaveformTest.cpp`